client/reader/update-notice/index.jsx

```diff
@@ -60,4 +49,8 @@ var UpdateNotice = React.createClass( {
 	}
 } );
 
-module.exports = UpdateNotice;
+export default connect(
```

Comment:
Ah, good catch, thanks. `onClick` really is a function, so I'll add `{ pure: false }`.